src/lib/util.js

```diff
@@ -87,9 +83,9 @@ module.exports = {
             fse.removeSync(dirs.work);
         }
 
-        // remove .nui if empty and unused otherwise
-        if (dirExistsAndEmpty(dirs.dotNui)) {
-            fse.removeSync(dirs.dotNui);
+        // remove build dir if empty and unused otherwise
+        if (dirExistsAndEmpty(dirs.build)) {
+            fse.removeSync(dirs.build);
```

Comment:
If clean up fails, everything crashes? (Don't remember out of the top of my head which behavior we wanted).

Reply:
Yes, it errors out here, and will fail the particular test case (report as "error" instead of "failure").

This is the same as before this PR.

But this was actually an issue in `nui` before the recent refactoring here and pulling in the "new" code here in the aio cli plugin, as the error thrown here (and related code) was not caught properly and the tests silently aborted. Now it's reported as "error" for the test case plus the message is shown.